src/simpleNbox.cpp

```diff
@@ -220,7 +236,7 @@ void SimpleNbox::setData( const std::string &varName,
         }
         else if( varNameParsed == D_Q10_RH ) {
             H_ASSERT( data.date == Core::undefinedIndex() , "date not allowed" );
-            q10_rh = data.getUnitval(U_UNITLESS);
+            q10_rh[ biome ] = data.getUnitval(U_UNITLESS);
```

Comment:
So, something like the case in `getData()`, setting _any_ of these with a biome that doesn't already exist will create it without ensuring that it is created consistently across all the other variables.  Maybe it's harmless, maybe not.  I know that we do occasionally iterate over these maps indiscriminately.  Can we show that having a bogus biome in one or more of these arrays will never cause a problem?  Can we ensure that nobody somewhere down the line will create a situation where it _will_ cause a problem?

Reply:
> Maybe it's harmless, maybe not.

I'm fairly certain it's _not_ harmless. I do know there are some runtime checks that can throw errors about mismatched sizes, but it's not immediately obvious that biome specification is what causes them.

Reply:
It's worth noting that this particular line, along with other changes related specifically to Q10, are needed to make Hector behave as documented. Prior to this change, Hector actually had _absolutely no mechanism_ for setting or using biome-specific Q10, even though the INI file indicates that it can (not that the INI file is an authoritative source of documentation, as we've discussed before, but even so).